src/routes/CreateNewSafePage/steps/OwnersAndConfirmationsNewSafeStep.tsx

```diff
@@ -0,0 +1,305 @@
+        <Paragraph className={classes.marginBottom} color="primary" noMargin size="lg">
+          Your Safe will have one or more owners. We have prefilled the first owner with your connected wallet details,
+          but you are free to change this to a different owner.
+        </Paragraph>
+        <Paragraph color="primary" size="lg" className={classes.descriptionContainer}>
+          Add additional owners (e.g. wallets of your teammates) and specify how many of them have to confirm a
+          transaction before it gets executed. In general, the more confirmations required, the more secure is your
+          Safe.
+          <Link
+            href="https://help.gnosis-safe.io/en/articles/4772567-what-gnosis-safe- setup-should-i-use"
+            target="_blank"
+            className={classes.link}
+            rel="noreferrer"
+            title="Learn about which Safe setup to use"
+          >
+            <Text size="xl" as="span" color="primary">
+              Learn about which Safe setup to use
+            </Text>
+            <Icon size="sm" type="externalLink" color="primary" />
+          </Link>
+          . The new Safe will ONLY be available on <NetworkLabel />
+        </Paragraph>
+      </Block>
+      <Hairline />
+      <Row className={classes.header}>
+        <Col xs={3}>NAME</Col>
+        <Col xs={7}>ADDRESS</Col>
+      </Row>
+      <Hairline />
+      <Block margin="md" padding="md">
+        <Row className={classes.header}>
+          {owners.map(({ nameFieldName, addressFieldName }) => {
+            const hasOwnerAddressError = formErrors[addressFieldName]
+            const showDeleteIcon = addressFieldName !== 'owner-address-0' // we hide de delete icon for the first owner
+
+            const handleScan = (address: string, closeQrModal: () => void): void => {
+              createSafeForm.change(addressFieldName, address)
+              closeQrModal()
+            }
+
+            return (
+              <Fragment key={addressFieldName}>
+                <Col xs={3}>
+                  <Field
+                    className={classes.ownerNameField}
+                    component={TextField}
+                    name={nameFieldName}
+                    placeholder="Owner Name"
+                    text="Owner Name"
+                    type="text"
+                    validate={minMaxLength(0, 50)}
+                    testId={nameFieldName}
+                  />
+                </Col>
+                <Col xs={7}>
+                  <AddressInput
+                    fieldMutator={(address) => {
+                      createSafeForm.change(addressFieldName, address)
+                      const addressName = addressBook[address]?.name
+                      if (addressName) {
+                        createSafeForm.change(nameFieldName, addressName)
+                      }
+                    }}
+                    // eslint-disable-next-line
+                    // @ts-ignore
```

Comment:
Why `@ts-ignore`?

Reply:
I will update the AddressInputProps to remove that `@ts-ignore`